.github/workflows/main-cmake-spc.yml

```diff
@@ -333,7 +328,7 @@ jobs:
             -DZLIB_USE_LOCALCONTENT=OFF \
             -DHDF5_ENABLE_MIRROR_VFD:BOOL=ON \
             -DHDF5_ENABLE_DIRECT_VFD:BOOL=ON \
-            -DH5_NO_DEPRECATED_SYMBOLS:BOOL=OFF \
+            -DHDF5_ENABLE_DEPRECATED_SYMBOLS:BOOL=OFF \
```

Comment:
Is this changing the default to disable deprecated symbols?

Reply:
It is using the correct name of the option to disable deprecated symbols.

Reply:
The intention of the job was to build develop branch with no deprecated functions. The default is ON, so yes this is changing the option from the default to OF so as to disable using deprecated functions.

Reply:
Because DH5_NO_DEPRECATED_SYMBOLS is not an option name

Reply:
Was the intent always to disable deprecated symbols for this build? If so why wasn't it set to BOOL=ON before?

Reply:
No I am not. The wrong name was used! Also the option should not have been in the other jobs.

Reply:
It may have used the wrong name but it by trying to set "no deprecated symbols" to OFF it was trying to do the opposite of what it does now. Even if it didn't work before, the intent in the code was clear, and it has been reversed. If that's what it needs to be that's fine, but it's going beyond a name change.

Reply:
But it didn't use no_deprecate. It was set to off.

Reply:
Are you saying that all the jobs were intended to disable deprecated symbols, but they mixed up the variable name and reversed the setting simultaneously?